package.json

```diff
@@ -1,54 +1,33 @@
 {
-  "name": "oc-k6",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "author": {
-	"name": "Florian Schade",
-	"email": "fschade@owncloud.com",
-	"url": "https://owncloud.com"
-  },
+  "version": "2.0.0-rc.2",
+  "private": true,
+  "homepage": "https://github.com/owncloud/cdperf",
+  "license": "AGPL-3.0",
+  "author": "ownCloud GmbH <devops@owncloud.com>",
   "scripts": {
-	"build": "rm -rf ./run && rollup -c",
-	"build:w": "rollup -c -w",
-	"clean": "rm -rf ./tests",
-	"lint": "eslint './src/**/*.ts'",
-	"lint:f": "yarn lint --fix",
-	"depcheck": "depcheck"
+    "build": "turbo run build --no-daemon",
```

Comment:
![image](https://user-images.githubusercontent.com/84779829/229075926-8a8a616d-68dc-4795-aa48-cd725fcdcb44.png)

to get `artifacts/*.js` I need:
- have node "18.9.1" and higher
- run `pnpm install` and `pnpm build`
After that I get `packages/k6-tests/artifacts/*.js`
and I can run test as: `k6 run packages/k6-tests/artifacts/koko-010-login.js` 
run `k6 run artifacts/koko-010-login.js` leads to error  `The moduleSpecifier "artifacts/koko-010-login.js" couldn't be found on local disk.`


Should we mention on READMI about: 
- node version
- running `pnpm` commands

Reply:
yes and no, in the end we should, but the instruction may change from rc to rc.
you have a second to chat in zoom? I need to see why it fails at your machine.

Reply:
thanks, issue resolved, you have to run the following steps to (re)-build the artifacts:

```
pnpm install
pnpm build

k6 run packages/k6-tests/artifacts/*
```

please only run those steps from the root, the mono-repos scripts take care of its packages on it's own.
I document everything as soon as we decided the nitty gritty details.

for the moment.....

https://github.com/owncloud/cdperf/blob/31b4d8952492c5de7eba11ddd43d359fb9bd356c/package.json#L8

... is a good starting point.

We ship all artifacts by default as part of the repository, technically no rebuild is needed.... but from the developer perspective, sure, we need it.

closing for now since the issue seems to be resolved